secretmanager/quickstart.rb

```diff
@@ -0,0 +1,72 @@
+# Copyright 2020 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def quickstart i_project_id: nil, i_secret_id: nil
+  # [START secretmanager_quickstart]
+  # Import the Secret Manager client library.
+  require "google/cloud/secret_manager"
+
+  # GCP project in which to store secrets in Secret Manager.
+  project_id = "YOUR_PROJECT_ID"
+
+  # ID of the secret to create.
+  secret_id = "YOUR_SECRET_ID"
+
+  # [END secretmanager_quickstart]
+  project_id = i_project_id if i_project_id
+  secret_id = i_secret_id if i_secret_id
```

Comment:
Is there a reason why we're using this technique rather than commenting out the initialization statements (as we do in the snippets.rb examples and in the examples for other services)?